pool/connection_pool_test.go

```diff
@@ -315,17 +315,17 @@ func TestConnectionHandlerOpenUpdateClose(t *testing.T) {
 	require.Nilf(t, err, "fail to set roles for cluster")
 
 	h := &testHandler{}
-	poolOpts := connection_pool.OptsPool{
+	poolOpts := pool.OptsPool{
```

Comment:
Shouldn't we rename `OptsPool` to `Opts`?

Reply:
It's a good idea, updated.